src/server/acl/acl_family.cc

```diff
@@ -193,4 +225,8 @@ void AclFamily::Register(dfly::CommandRegistry* registry) {
 
 #undef HFUNC
 
+void AclFamily::Init(std::vector<facade::Listener*> listeners) {
```

Comment:
why not const& and than copy here?

Reply:
I like to delegate this to the callers. An example is:

std::vector<...> tmp;
Init(std::move(tmp)); //init moves internally so in total 2 moves

If we do what you suggest we end up with 1 move(fine const& will work) and then a deep copy. 

Generally, I like to:
1. Accept by value an delegate the choice to the user.
2. Accept by && only if the function is supposed to be used as a sink.